practices/pianissimo-pp/practica2/niveles-linguisticos-2.py

```diff
@@ -0,0 +1,175 @@
+import warnings
+warnings.filterwarnings("ignore")
+
+# ---
+# Ejercicio 1
+# ---
+
+import json
+from unidecode import unidecode
+
+# Leemos el archivo corpus_otomi
+
+data = []
+with open("./practices/pianissimo-pp/practica2/corpus_otomi", "r") as file:
```

Comment:
Como subiste el archivo del corpus basta con dejarlo asi:

```suggestion
with open("corpus_otomi", "r") as file:
```